atlasdb-config/src/main/java/com/palantir/atlasdb/factory/TransactionManagers.java

```diff
@@ -1267,7 +1268,7 @@ private static LeaderTimeGetter getLeaderTimeGetter(
             return new LegacyLeaderTimeGetter(namespacedConjureTimelockService);
         }
 
-        LeaderTimeCoalescingBatcher batcher =
+        ReferenceTrackingWrapper<LeaderTimeCoalescingBatcher> batcher =
```

Comment:
I think we should increase the reference count here, since we are decreasing it on close. Also the same for the other two batchers above

Reply:
I was thinking of adding it to internal proxy where the constructors live but this is fine too